presto-docs/src/main/sphinx/connector/inmemory.rst

```diff
@@ -0,0 +1,55 @@
+==================
+InMemory Connector
+==================
+
+The InMemory connector stores all data and metadata in RAM on workers
+and both are discarded when Presto restarts.
+
+.. warning::
+
+    This connector is in early experimental stage it is not recommended
+    to use it on a production environment.
+
+.. warning::
+
+    This connector does not support ``DROP TABLE`` or any other method
+    to release held resources. Once table is populated it will consume
+    memory until next Presto restart.
+
+.. warning::
+
+    When one worker fails/restarts all data that were stored in it's
+    memory will be lost forever. In other words in case of worker
+    restart there will be partial data loss.
```

Comment:
"partial data loss" -> please explain whether users are warned in any way. Or is this silent data loss?